src/main/java/assignment2/utils/InputOutputUtil.java

```diff
@@ -0,0 +1,27 @@
+package assignment2.utils;
+
+import assignment2.exceptions.InvalidException;
+import assignment2.utils.InputValidators.YesOrNoValidate;
+import java.util.Scanner;
+
+public class InputOutputUtil {
+
+  private static final Scanner sc = new Scanner(System.in);
+
+  public static String input() { return sc.nextLine().trim(); }
+
+  public static void output(String Text) {
+    System.out.println(Text);
+  }
+
+  public static char getYesNoInput() throws InvalidException {
+    String input = InputOutputUtil.input();
```

Comment:
you can call input() method directly without class reference as it is present in the same class.